torch/storage.py

```diff
@@ -958,13 +1049,13 @@ def _setitem(self, idx, value):
             tmp_tensor = torch.tensor(
                 [], dtype=tmp_dtype, device=self._untyped_storage.device
             )
-            tmp_tensor.set_(
+            tmp_tensor.set_(  # type: ignore[call-overload]
```

Comment:
All of these `type: ignore`s arise because I have removed `TypedStorage` from the public API type annotations (as it is to be deprecated).